package.json

```diff
@@ -7,7 +7,7 @@
   "main": "./dist/demosplan-ui.umd.js",
   "dependencies": {
     "@braintree/sanitize-url": "^6.0.1",
-    "@demos-europe/demosplan-utils": "^0.0.3",
+    "@demos-europe/demosplan-utils": "github:demos-europe/demosplan-js-utils#main",
```

Comment:
Are we going to keep it like that or change it to the local npm package later?

Reply:
i think it makes sense to leave it like so until we release the next demosplan-core version.

Reply:
This has to be changed to the next released versions after we tested everything and think its "stable".
I just want to avoid releasing every couple of hours.